README.md

```diff
@@ -93,4 +102,4 @@ HOTKEYS
 
 Note
 ----
-Not all hotkeys work between Spotify, iTunes, Winamp, foobar2000, and VLC.
+Not all hotkeys work between Spotify, iTunes, Winamp, foobar2000,mps-youtube and VLC.
```

Comment:
Missing a space.
"Not all hotkeys work between Spotify, iTunes, Winamp, foobar2000, VLC, and mps-youtube.